sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClient.java

```diff
@@ -371,13 +372,36 @@ public String getEntityPath() {
             return fluxError(logger, new NullPointerException("'scheduledEnqueueTime' cannot be null."));
         }
 
-        return createBatch().flatMapMany(messageBatch -> {
-            messages.forEach(message -> messageBatch.tryAdd(message));
-            return getSendLink().flatMapMany(link -> connectionProcessor
-                .flatMap(connection -> connection.getManagementNode(entityName, entityType))
-                .flatMapMany(managementNode -> managementNode.schedule(messageBatch.getMessages(), scheduledEnqueueTime,
-                    messageBatch.getMaxSizeInBytes(), link.getLinkName(), transactionContext)));
-        });
+        return getSendLink().flatMapMany(link -> link.getLinkSize()
```

Comment:
Why did we duplicate createBatch() logic? By default it uses the default batch size.

Reply:
I was checking size on the link but changed it to use default size.